### ISSUE TITLE
Register functions not working

### ISSUE THREAD

**Issue by eurobob** [posva/vue-mdl#19]
I'm using browserify, and I can require vue-mdl fine, but when I try to register components I get this error:

```Uncaught TypeError: vmdl.registerAll is not a function```

**Comment by andrewex** [comment]
I just ran into the same error. Looking at the vmdl object in the console I noticed the register and registerAll functions are nested under the default object. Check out the code below that is working for me. Also make sure to include the Material Lite dependencies.

var vmdl = require('vue-mdl');
var Vue = require('vue');

vmdl.default.registerAll(Vue);

var checkbox = vmdl.default.components['mdl-checkbox'];
var badge = vmdl.default.directives['mdl-badge'];

export default {
  components: {
    mdlCheckbox: checkbox
  },
  directives: {
    mdlBadge: badge
  }
}
